scripts/local/apply-all-migrations.sh

```
@@ -1,6 +1,31 @@
+    printf "${DANGER}Invalid argument: '$TARGET_ENV'. Usage: $0 [-psh] [dev|staging|production]${NC}\n"
+    exit 1
+    ;;
+esac
+echo "Migrating database for environment: $ENV_FILE"
+
+if [ ! -f "$ENV_FILE" ]; then
+  err_env_missing
+  exit
+fi
+
+export $(grep -v '^#' $ENV_FILE | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)